src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationCriminalComplianceChecks.cshtml

```diff
@@ -119,178 +51,28 @@
-                                           @radioCheckedInProgress />
-
-                                    <label class="govuk-label govuk-radios__label" for="InProgress">
-                                        In progress
-                                    </label>
-
-                                </div>
-
-                                <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-InProgress">
-                                    <div class="govuk-character-count" data-module="character-count" data-maxwords="150">
-                                        <div class="govuk-form-group">
-                                            <label for="OptionInProgressText" class="govuk-label">
-                                                Add comments (optional)
-                                            </label>
-                                            <textarea class="govuk-textarea js-character-count"
-                                                      id="OptionInProgressText"
-                                                      name="OptionInProgressText"
-                                                      rows="5"
-                                                      maxlength="10000">@Model.OptionInProgressText</textarea>
-                                        </div>
-
-                                        <span id="word-count-info-InProgress" class="govuk-hint govuk-character-count__message" aria-live="polite"></span>
-                                    </div>
-                                </div>
-                            }
-                        </div>
-                    </fieldset>
-                </div>
-
-                <button class="govuk-button" type="submit">Save and continue</button>
-            </form>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <form class="js-disable-on-submit" asp-action="EvaluateCriminalCompliancePage" asp-controller="RoatpOrganisationCriminalComplianceChecks">
+                    @Html.HiddenFor(x => x.ApplicationId)
+                    @Html.HiddenFor(x => x.PageId)
+                    @Html.HiddenFor(x => x.Caption)
+                    @Html.HiddenFor(x => x.Heading)
+                    @Html.HiddenFor(x => x.NoSelectionErrorMessage)
+                    @Html.HiddenFor(x => x.Ukprn)
+                    @Html.HiddenFor(x => x.ApplyLegalName)
+                    @Html.HiddenFor(x => x.QuestionText)
+                    @Html.HiddenFor(x => x.ComplianceCheckAnswer)
+                    @Html.HiddenFor(x => x.ComplianceCheckQuestionId)
+                    @Html.HiddenFor(x => x.FurtherInformationAnswer)
+                    @Html.HiddenFor(x => x.FurtherInformationQuestionId)
+                    @{ await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayApprovalSection.cshtml"); }
```

Comment:
Use the partial tag helper as Html.RenderPartialAsync is deprecated in ASP.NET Core